packages/venice-icons/config.json

```diff
@@ -1545,6 +1545,20 @@
       "search": [
         "warning-outline"
       ]
+    },
```

Comment:
Question: Why are u adding a colorful image on `venice-icons`? Can u add an evidence on the PR's description? Wouldn't it be better to add directly to our S3 bucket?